### PR TITLE
add core v1 helper ut

### PR DESCRIPTION
**Pull request by CoderTH**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by CoderTH** [comment]
> /ok-to-test /release-note-none

Could you review it for me ？

File: pkg/apis/core/v1/helper/helpers_test.go [hunk 1]
```diff
@@ -806,3 +808,100 @@ func TestLoadBalancerStatusEqual(t *testing.T) {
...
+				t.Errorf("Expected %v for resource name %s, got %v", tc.expected, tc.name, result)
+			}
+		})
+	}
+}
+
+func TestIsHugePageResourceName(t *testing.T) {
+	type args struct {
+		name v1.ResourceName
+	}
```
Comment (pacoxu) [inline pkg/apis/core/v1/helper/helpers_test.go:850]: Why define `args` struct here? It is not used in the test.

Comment (CoderTH) [reply pkg/apis/core/v1/helper/helpers_test.go]: done

**Comment by pacoxu** [comment]
/priority backlog
/triage accepted

coverage change with this PR: from 56.6% to 61.4%.
/cc liggitt

```
        k8s.io/kubernetes/pkg/apis/core/v1/helper       coverage: 61.4% of statements
```
/lgtm

**Comment by CoderTH** [comment]
@liggitt  Hello , can you help me review the code

**Comment by CoderTH** [comment]
@dims @smarterclayton 
hi,this pr has not been updated for a long time, could you please help review it ?

**Pull request closed without merging**